src/Upgrade/Application/Strategy/Composer/Fixer/FeaturePackageFixerStep.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+declare(strict_types=1);
+
+namespace Upgrade\Application\Strategy\Composer\Fixer;
+
+use Upgrade\Application\Adapter\PackageManagerAdapterInterface;
+use Upgrade\Application\Dto\StepsResponseDto;
+use Upgrade\Application\Strategy\FixerStepInterface;
+use Upgrade\Domain\Entity\Collection\PackageCollection;
+use Upgrade\Domain\Entity\Package;
+
+class FeaturePackageFixerStep implements FixerStepInterface
+{
+    /**
+     * @var string
+     */
+    protected const PATTERN = '/(?<features>[-\w]+-feature\/[-\w]+).+conflicts.+/';
```

Comment:
The first part is not needed as regex Wildcard
Is always spryker-feature, right?

Reply:
can be not. Current logic for feature packages to split them.
What do you mean first part?

Reply:
Read My comment again, I Noted the exact part

Reply:
maybe yes. but it doesn't change anything.

Reply:
`[-\w]+-feature` => why the wildcard, why not `spryker-feature` ?

Reply:
changed